docs/api/transport.md

```diff
@@ -57,6 +58,11 @@ Highlights on transporting functions are:
 - Closure cannot be transported, but you won't get error when transporting a function. Instead, you will get runtime error complaining a variable (from closure) is undefined when you can the function later.
 - `__dirname` / `__filename` can be accessed in transported function, which is determined by `origin` property of function. By default `origin` property is set to current working directory.
 
+### <a name="transporting-built-in"></a> Transporting JavaScript built-in object
+JavaScript standard built-in objects in [_builtInTypeWhitelist](./../../lib/transport/transport.ts) can be transported among napa workers transparently. JavaScript Objects with properties in these types are also able to be transported. Please refer to [transportBuiltinObjects](./../../test/transport-test.ts) for detail.
```

Comment:
It's better not to expose private members in documentation, since it may change. You can list the types here, and update the doc when we update the white list.

Reply:
[transportBuiltinObjects] => [unit tests]